clients/linux/src/filetree.rs

```diff
@@ -347,10 +347,52 @@ impl FileTree {
         self.tree.get_selection().get_selected_rows()
     }
 
+    pub fn vec_into_tree(
+        &self,
+        metadatas: &[DecryptedFileMetadata],
+        parent_metadata: &DecryptedFileMetadata,
+        parent_iter: &TreeIter,
+    ) -> LbResult<()> {
```

Comment:
This function will always return `Ok(())`, so I don't see the need for a return type. Am I missing something?

Reply:
You're right!